Extensions/classic_header.js

```diff
@@ -100,15 +112,47 @@ XKit.extensions.classic_header = new Object({
 			XKit.extensions.classic_header.show_blogs();
 		}
 		if (XKit.extensions.classic_header.preferences.fixed_width.value === true) {
-			XKit.tools.add_css(" @media screen and (min-width: 900px){.l-header {max-width: 900px!important;}}", "classic_header_fixed_width");
+			$( function() {
+				var cwidth = $(".l-content").outerWidth() + 31;
+				if (cwidth < 816) { return; }
+				var lpad = 2;
+				if (XKit.extensions.tweaks.preferences.old_sidebar_width.value && $("#right_column").length > 0) {
+					lpad += 75;
+				}
+				XKit.tools.add_css(
+				"@media screen and (min-width: " + cwidth + "px)" +
```

Comment:
It would be nice if this section were wrapped in a more aesthetically pleasing fashion like
```
"@media blah () {" +
"  .l-header {" +
"    max-width: " + cwidth + " blah;" +
"  }" +
"}",
```